app/src/main/java/com/hedvig/app/feature/home/ui/HomeFragment.kt

```diff
@@ -52,6 +53,13 @@ class HomeFragment : Fragment(R.layout.home_fragment) {
             marketManager
         )
 
+        binding.swipeToRefresh.apply {
+            setOnRefreshListener {
+                model.load()
```

Comment:
You want to set isRefreshing = true here right? And when data is loaded set it to false.